### ISSUE TITLE
[SUGGESTION] <New Method to set the Inventory Item>

### ISSUE THREAD

**Issue by NickzMC** [oraxen/oraxen#526]
### I have checked that...

- [X] ...such a feature does not exist already
- [X] ...such a feature request has not been submitted already

### Description

I would like to have the option to change the /o inv item from the GUI in the same config where I set the items.

e.g. the first item could be taken from the config by default

### Reason

When I create packs I would like to set the show item, unfortunately this is never possible because I have to add another config although this would only be a small setting.

**Comment by Boy0000** [comment]
you can changhe the item displayed in /oraxen inv inside settings.yml at the bottom
If u mean having it just show the first item in the yml file perhaps but at best this would be low priority to add

**Comment by NickzMC** [comment]
yes, but i it would be better if its in the working files like this:
![Unbenannt](https://user-images.githubusercontent.com/62188951/197328072-c6cf98bb-7c11-478a-95c1-3ee307f4c86b.PNG)
would make more sense from my point of view :D.
does not have to have a priority, would only be nice if it would work so too

**Comment by Boy0000** [comment]
oraxen would read it as item configs
tldr this is just too much work tbh as it needs to redo part of the core system, just so its not in settings.yml
